LibVLCSharp/Shared/LibVLC.cs

```diff
@@ -596,9 +573,11 @@ public MediaDiscoverer.Description[] MediaDiscoverers(MediaDiscoverer.Category c
         {
             var arrayResultPtr = IntPtr.Zero;
             var count = Native.LibVLCMediaDiscovererListGet(NativeReference, category, ref arrayResultPtr);
-
+#if NETSTANDARD1_1
+            if (count == 0) return new MediaDiscoverer.Description[0];
+#else
             if (count == 0) return Array.Empty<MediaDiscoverer.Description>();
-
+#endif
```

Comment:
Below, you replaced with `if (count == 0) return new RendererDescription[0];` for all frameworks, which is the most compatible.
Can you use the same fix for both?

Reply:
I found some articles saying Array.Empty caches the new T[0] call for future use : http://justinvp.com/2015/07/20/array-empty/ 
You can keep that for perf if you want, but chose only one solution.

Reply:
> chose only one solution.

If I want to keep using `Array.Empty` where possible, I can't choose "only one solution". 
It's either `new [0]` or a mix of both depending on what is available.

Reply:
OK, but then you could use the same #ifdef logic everywhere